person.rb

```diff
@@ -27,4 +28,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] `add_rental method should receive `book` and `date` as parameter. Also the person itself should be added as parameter.
`
def add_rental(book, date)
...
end
`

Reply:
Done